### ISSUE TITLE
how to get start date and end date of month 

### ISSUE THREAD

**Issue by AnjaniJoshi** [prolificinteractive/material-calendarview#875]

**Comment by osrl** [comment]
It's not related to this library but here is your answer:

```
ClendarDay d = // your CalendarDay Instance
CalendarDay first = d.date.with(TemporalAdjusters.firstDayOfMonth())
CalendarDay last = d.date.with(TemporalAdjusters.lastDayOfMonth())

**Comment by ankitk93** [comment]
@osrl can you please elaborate the code, not able to get it.

**Comment by osrl** [comment]
maybe this will make it clear for you @ankitk93 

```
LocalDate date = // localdate instance
LocalDate first = date.with(TemporalAdjusters.firstDayOfMonth())
LocalDate last = date.with(TemporalAdjusters.lastDayOfMonth())
```

you can read from [here](https://www.threeten.org/threetenbp/apidocs/org/threeten/bp/temporal/TemporalAdjusters.html) and [here](https://www.threeten.org/threetenbp/apidocs/org/threeten/bp/LocalDate.html#with(org.threeten.bp.temporal.TemporalAdjuster))

**Comment by ankitk93** [comment]
Yeah but this will not work when you change the month and try to get the First date and last date of that selected month. Because the MaterialCalenadarView's on monthChangeListener will not accept those values.So no use i guess
